engine/gamesys/src/gamesys/components/comp_label.cpp

```diff
@@ -73,14 +73,18 @@ namespace dmGameSystem
         dmRender::HMaterial         m_Material;
         dmRender::HFontMap          m_FontMap;
 
+        float                       m_Leading;
+        float                       m_Tracking;
+
         const char*                 m_Text;
 
         uint16_t                    m_ComponentIndex;
         uint16_t                    m_Enabled : 1;
         uint16_t                    m_AddedToUpdate : 1;
         uint16_t                    m_UserAllocatedText : 1;
         uint16_t                    m_ReHash : 1;
-        uint16_t                    m_Padding : 4;
+        uint16_t                    m_LineBreak : 1;
+        uint16_t                    m_Padding : 3;
```

Comment:
Old bug: There are 16 bits in an uint16_t.
The old padding assumed it was an 8 bit variable.
Also, no need for a name:
```
uint16_t :11;
```

Reply:
Nice catch! I missed that!